integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -1798,6 +1798,34 @@ msgstr "Vierte Woche"
 msgid "Last week"
 msgstr "Letzte Woche"
 
+#: cms/forms/contacts/contact_form.py
+msgid "Location cannot be empty."
+msgstr "Ort darf nicht leer sein."
+
+#: cms/forms/contacts/contact_form.py
+msgid ""
+"E-mail address does not match with that of the selected location. Please "
+"check again."
+msgstr ""
+"E-mail Adresse stimmt nicht mit der vom ausgewählten Ort. Bitte prüfen Sie "
+"noch mal."
+
+#: cms/forms/contacts/contact_form.py
+msgid ""
+"Phone number does not match with that of the selected location. Please check "
+"again."
+msgstr ""
+"Telefonnummer stimmt nicht mit der vom ausgewählten Ort. Bitte prüfen Sie "
+"noch mal."
+
+#: cms/forms/contacts/contact_form.py
+msgid ""
+"Website URL does not match with that of the selected location. Please check "
+"again."
+msgstr ""
+"Website URL stimmt nicht mit der vom ausgewählten Ort. Bitte prüfen Sie noch "
+"mal."
```

Comment:
I advocate against these messages altogether, but if we keep them, I suggest the following wording:
```suggestion
#: cms/forms/contacts/contact_form.py
msgid ""
"E-mail address does not match with that of the selected location. Please "
"check again."
msgstr ""
"Die E-mail Adresse stimmt nicht mit der vom ausgewählten Ort überein. Bitte "
"überprüfen Sie sie erneut."

#: cms/forms/contacts/contact_form.py
msgid ""
"Phone number does not match with that of the selected location. Please check "
"again."
msgstr ""
"Die Telefonnummer stimmt nicht mit der vom ausgewählten Ort überein. Bitte "
"überprüfen Sie sie erneut."

#: cms/forms/contacts/contact_form.py
msgid ""
"Website URL does not match with that of the selected location. Please check "
"again."
msgstr ""
"Die Website URL stimmt nicht mit der vom ausgewählten Ort überein. Bitte "
"überprüfen Sie sie erneut."
```